nhd/Node.py

```diff
@@ -391,7 +398,16 @@ def InitNics(self, labels):
 
                 (ifname, vendor, mac, speed, numa_node, pciesw, card, port) = (p[4], p[5], p[6], p[7], int(p[8]), int(p[9],16), int(p[10],16), int(p[11]))
 
+                extra_label = ''
+                sriov = False
+                try:
+                    extra_label = p[12]
```

Comment:
I think this is the main question - how exactly is this extra 'bkp' getting applied to the label.  That implies a change to NFD is also needed to add that label, correct?  
Since we already have assumptions in place for associating port pairs, can we simply infer the backup ports without needing the label?  example:
non-VF ports -> f0 is primary, f1 is backup
VF ports - count total port, first half are primary, second half are backup.
It would only take a few lines in InitNics to do that.
I think to implement in NFD, it would just need to make the same set of assumptions, so best just to keep that all in one place.

Reply:
Hi. Originally decison I arrived at with @bendavis0 was to keep labeling out of NHD and let it focus on scheduling. This will allow  any potential users to write their own "bkp" interface assignment logic without changing NHD code.